<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana</class>
 <widget class="QMainWindow" name="ventana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kinoso v0.8</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>1031</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>550</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>550</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>550</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>550</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>550</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>550</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>550</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE10">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>550</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE11">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>550</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE12">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>550</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE13">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>550</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE14">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>550</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>550</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar Combinaci√≥n</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>550</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Repetidas</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>701</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="botonCambioSorteo">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cambiar Sorteo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>726</x>
      <y>560</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bauhaus 93</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kinoso 0.8</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionIngresar_sorteo"/>
    <addaction name="separator"/>
    <addaction name="actionMostrar"/>
    <addaction name="actionModificar_sorteo"/>
    <addaction name="actionBorrar_sorteo"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuCombinatoria">
    <property name="title">
     <string>Combinatoria</string>
    </property>
    <addaction name="actionCombinaciones"/>
    <addaction name="actionProbabilidades"/>
    <addaction name="actionMigrar_base_a_texto"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuCombinatoria"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionIngresar_sorteo">
   <property name="text">
    <string>Ingresar sorteo</string>
   </property>
  </action>
  <action name="actionModificar_sorteo">
   <property name="text">
    <string>Modificar sorteo</string>
   </property>
  </action>
  <action name="actionBorrar_sorteo">
   <property name="text">
    <string>Borrar sorteo</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionMostrar">
   <property name="text">
    <string>Mostrar</string>
   </property>
  </action>
  <action name="actionCombinaciones">
   <property name="text">
    <string>Combinaciones</string>
   </property>
  </action>
  <action name="actionProbabilidades">
   <property name="text">
    <string>Probabilidades</string>
   </property>
  </action>
  <action name="actionMigrar_base_a_texto">
   <property name="text">
    <string>Migrar base a texto</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
